src/perturbopy/postproc/utils/constants.py

```diff
@@ -14,7 +14,7 @@
                       'J': ['j', 'joule', 'joules'],
                       'Ry': ['ry', 'rydberg', 'rydbergs']}
 
-energy_units_vals = {'eV': (2.7211396132, 1), 'Ha': (1, 0), 'J': (4.359748199, -18), 'Ry': (0.5, 0)}
+energy_units_vals = {'eV': (2.7211396132, 1), 'Ha': (1, 0), 'J': (4.359748199, -18), 'Ry': (2.0, 0)}
```

Comment:
What is the reason of this change?

Reply:
The conversion was wrong before.

energy_conversion_factor('Ry', 'eV')
after the fix it gives:
>>> ppy.utils.constants.energy_conversion_factor('Ry', 'eV')
13.605698066

Before the fix, there was a wrong number